python/lsst/the/monster/splinecolorterms.py

```diff
@@ -333,11 +333,11 @@ def apply(self, source_color_flux_1, source_color_flux_2, source_flux):
 
         mag_color = mag_1 - mag_2
 
-        model = self.spline.interpolate(mag_color)
-        model -= self.flux_offset/source_flux
+        target = self.spline.interpolate(mag_color) * source_flux
```

Comment:
are you sure target = model * source_flux 
based off the cost function https://github.com/lsst-dm/the_monster/blob/08b3df7e1f1a56311da2aa0d2f48d0b714a39770/python/lsst/the/monster/splinecolorterms.py#L206

I would have assumed target_flux = source_flux / model